src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.GenerateAssemblyInfo.targets

```diff
@@ -48,8 +49,21 @@ Copyright (c) .NET Foundation. All rights reserved.
           DependsOnTargets="PrepareForBuild;CoreGenerateAssemblyInfo"
           Condition="'$(GenerateAssemblyInfo)' == 'true'" />
 
+  <Target Name="AddSourceRevisionToInformationalVersion"
+          DependsOnTargets="InitializeSourceControlInformation"
```

Comment:
This should probably also depend on the `GetAssemblyVersion`, which is where `InformationalVersion` is set by default.  It currently works because of the ordering of `DependsOnTargets` for `GetAssemblyAttributes`, but it's probably better to be more explicit.

Reply:
OK.